manifest.json

```diff
@@ -6,13 +6,12 @@
         "en": "LibreOffice-based online office suite with collaborative editing",
         "fr": "Suite office en ligne et collaborative, basée sur LibreOffice"
     },
-    "version": "6.4.10~ynh2",
+    "version": "21.11.0.6~ynh1",
```

Comment:
As I said in the original PR, I don't know what will happen when the package _coolwsd_ will be updated on the collabora deb, but I feel like a simple `apt upgrade` will upgrade the app (but perhaps the package is marked "on hold" by Ynh). I don't know how Yunohost apps deal with this kind of situation.

Reply:
you maybe right, at least user will be informed from the readme of the version being installed.

Reply:
@Taekiro indeed the virtual packages generated by YunoHost mark the dependencies as "on hold".

Reply:
OK, I didn't knew that. How to upgrade the app then? Does a simple change of the version in the manifest trigger the apt upgrade on the held package?